.github/workflows/pr-check-e2e.yml

```diff
@@ -28,8 +28,8 @@ jobs:
       - name: Install dependencies
         run: yarn install
 
-      - name: Run application and cypress/accessibility tests
```

Comment:
Now, we can split pa11y tests from cypress tests because seeding data doesn't depend on cypress tests.